integration/fuse/src/main/java/alluxio/fuse/auth/SystemUserGroupAuthPolicy.java

```diff
@@ -13,100 +13,60 @@
 
 import alluxio.AlluxioURI;
 import alluxio.client.file.FileSystem;
-import alluxio.exception.AlluxioException;
 import alluxio.fuse.AlluxioFuseFileSystemOpts;
 import alluxio.fuse.AlluxioFuseUtils;
-import alluxio.grpc.SetAttributePOptions;
 import alluxio.jnifuse.AbstractFuseFileSystem;
 import alluxio.jnifuse.struct.FuseContext;
 
-import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
-import com.google.common.cache.LoadingCache;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.google.common.base.Preconditions;
 
-import java.io.IOException;
-import java.util.concurrent.ExecutionException;
+import java.util.Optional;
 
 /**
- * Default Fuse Auth Policy.
+ * The system user group authentication policy that always set the user group
+ * to the actual end user.
+ * Note that this may downgrade the performance of creating file/directory.
  */
-public final class SystemUserGroupAuthPolicy implements AuthPolicy {
-  private static final Logger LOG = LoggerFactory.getLogger(
-      SystemUserGroupAuthPolicy.class);
+public final class SystemUserGroupAuthPolicy extends LaunchUserGroupAuthPolicy {
```

Comment:
this inheritance relation looks a little strange,

Reply:
Yeah, it's more for reusing the code, the method in LaunchUserGroupAuthPolicy can be reused, i will think of whether there are better ways to do the same thing